### ISSUE TITLE
Build not triggered when tag created

### ISSUE THREAD

**Issue by bb-tle** [Eernie/bitbucket-webhooks-plugin#73]
Pushing to Bitbucket (version 5.3.1) works nicely (i.e., triggering a build); however, if a tag is created, nothing happens. The job is created in Blue Ocean (pipeline). Thank you. T. Le

**Comment by casz** [comment]
As I recall Bitbucket Branch Source plugin has yet to support building from tags 😐

**Comment by Eernie** [comment]
Correct, This plugin wil send an event, but the if you're using the Bitbucket Branch Source plugin as the receiving end, it won't do anything.

**Comment by bb-tle** [comment]
Thank you, casz and Eernie.

**Comment by bb-tle** [comment]
Today I updated Bitbucket Branch Source plugin to 2.2.6, which is supposed to support building from tags. However it did not build when a new tag was created.
I then turned on access logging for Jenkins and noticed that 1) when "On Push" and "Tag Created" were both selected from Post Webhooks, I could see in the Jenkins access log the line "POST /bitbucket-scmsource-hook/notify HTTP/1.1" 200 0 "-" "Apache-HttpClient/4.5.1 (Java/1.8.0_102)" showing Post Webhooks plugin informed Jenkins of the change. However, if 2) I only enabled "Tag Created" and performed a push, the above log entry did not appear, implying Post Webhooks did not communicate with Jenkins when on tagging was done.
In both cases, I modified a file in master branch, created a tag (annotated) and finally used command "git push --follow-tags" to push. Thank you in advance for your insight.
